From 7cd0009be5f548696b7f358cb442d557ace7dde3 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Wed, 24 Jan 2024 14:45:58 +0100
Subject: [PATCH 0583/1695] dts: stm32: share PMIC vdd3v3_usb for
 stm32mp257f-ev1

Share vdd3v3_usb from the PMIC in order to later remove
always-on property.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 133af10e88220e8a5b0212234e037c968c2a3e7b dts: stm32: share PMIC vdd3v3_usb for stm32mp257f-ev1]
Change-Id: Iabca50f82efa9d040dd67a87ae9b317a0a789049
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/353988
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index a68d4ffd3..6a73d2dc8 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -429,6 +429,10 @@
 		reg = <VOLTD_SCMI_STPMIC2_LDO2>;
 		voltd-supply = <&vdd_emmc>;
 	};
+	voltd-vdd3v3-usb {
+		reg = <VOLTD_SCMI_STPMIC2_LDO4>;
+		voltd-supply = <&vdd3v3_usb>;
+	};
 	voltd-vdd-sdcard {
 		reg = <VOLTD_SCMI_STPMIC2_LDO7>;
 		voltd-supply = <&vdd_sdcard>;
-- 
2.47.2

